front-end/js/pageOnLoad.js

```diff
@@ -100,6 +100,16 @@ function processRenderingParams() {
     }
 }
 
+// add the styles to the document
+function processStyles() {
+    var styles = d3
```

Comment:
you can check whether styles is an empty string, so you don't add one <style> element when there is no styles specified.